databook/background/Two-photon-calcium-imaging.md

```diff
@@ -52,21 +56,23 @@ brains. While wide-field microscopy can still be used on three-dimensional
 structures, contrast and resolution are hampered by scattering and out-of-focus
 fluorescence. Confocal imaging rejects out-of-focus light (i.e. provides optical
 sectioning), but excites large amount of tissue that is not being imaged, and
-also rejects photons emitted by the tissue being imaged but are scattered by the
-tissue. In contrast, two-photon excitation microscopy, or two-photon microscopy
-for short, uses two photons with longer wavelengths than the emitted light to
-excite a fluorophore {cite:p}`denk1990`. With two-photon excitation, the
-absorption is a nonlinear function of photon density, allowing microscopes to
-only excite a single voxel of tissue at a time. This improves our ability to
-image the activity of {term}`GECI` expressing neurons in intact tissue.
-
-There are limits to what we can see using two-photon miscroscopy, however. Two-photon excitation generates fluorescence primarily from the focal plane, but out-of-focus fluorescence from outside the focal plane will contaminate the view. This out-of-focus fluorescence increases as the focus is pushed deeper into tissue {cite:p}`takasaki`. This can be problematic when imaging densely expressed GECIs where there is fluoresence throughout the tissue, where the image quality degrades with imaging depth. However, using {term}`Cre line`s that limit the GECI expression only to deep neurons (e.g. Cre lines specific to layer 5 or layer 6 pyramidal neurons) can avoid this problem.
+also rejects light that is emitted by the tissue being imaged but is scattered by the
```

Comment:
let's keep the original version here.

Reply:
reverted, albeit with grammar/typo fixes